package.json

```diff
@@ -37,7 +37,6 @@
     "jsdoc-to-markdown": "^2.0.1",
     "mocha": "^3.2.0",
     "mochainon": "^1.0.0",
-    "wary": "^1.1.0"
```

Comment:
Fair enough. I had a local-branch in progress with changes related to these image-writing tests, but I'm happy to rebase after this PR has been merged. (and the removal of `wary` will actually make that easier)

> we don't need that in this case, as the output shouldn't change anyways

AFAIK the output-changing shouldn't matter (my local-branch actually removes the output files from git and creates and deletes them on-demand); I'd assumed `wary` was used so that all the "big writes" happen in a temporary directory, rather than on the local filesystem? \*shrug\*